testing/test_terminal.py

```diff
@@ -689,6 +689,32 @@ def test_three():
         )
         assert result.ret == 0
 
+    def test_keyword_with_private_node(self, pytester: Pytester) -> None:
```

Comment:
this test doesn't fail on main -- please write a test which demonstrates your change

Reply:
@asottile, I don't understand
>doesn't fail on main -- 
Can you please elaborate, so that I can make a change accordingly.

This is the output that I see when the test it run with 
 ```python
 result = pytester.runpytest(testpath)
 ```
 instead of the line at https://github.com/pytest-dev/pytest/pull/8648/files#diff-351b791015528a514e6df6c55de8f12848e272c705f6cfe2436bbeba4a3204d3R709
 
 output:
 ```
 E       Failed: nomatch: 'collected 5 items / 3 deselected / 2 selected'
E           and: '============================= test session starts =============================='
E           and: 'platform linux -- Python 3.7.10, pytest-6.3.0.dev434+gcb8f1d0b7.d20210509, py-1.10.0, pluggy-0.13.1'
E           and: 'rootdir: /tmp/pytest-of-rahul/pytest-20/test_keyword_with_private_node0'
E           and: 'collected 5 items'
E           and: ''
E           and: 'test_keyword_with_private_node.py .....                                  [100%]'
E           and: ''
E           and: '============================== 5 passed in 0.02s ==============================='
E       remains unmatched: 'collected 5 items / 3 deselected / 2 selected'
 ```
 
 I think the added test demonstrates the included change, please let me know if I am missing something.

Reply:
if I `git checkout origin/main -- src` your test still passes

Reply:
the test is incorrect - the intended trigger will not collect as test to begin with